Add organization flag to Create Project step

beritou opened the pull request:
Closes: #8

beritou committed:
Add organization flag to Create Project step

Closes: #8

marcoderama reviewed (approved):
But see comment https://github.com/lightbend/kalix-trial-typescript-shoppingcart/pull/6#pullrequestreview-1849242885 which would also apply here.

beritou committed:
Update Create Projects instructions with more detail

Add step to check organization name using "kalix organization list" and
instruct user to use obtained organization name in "kalix projects new" command.
